wd_helper.au3

```diff
@@ -3533,13 +3577,14 @@ Func __WD_GetLatestWebdriverInfo($aBrowser, $sBrowserVersion, $bFlag64)
 	Local $sURL = $aBrowser[0][$_WD_BROWSER_LatestReleaseURL]
 	Local $sRegex = $aBrowser[0][$_WD_BROWSER_LatestReleaseRegex]
 	Local $sNewURL = $aBrowser[0][$_WD_BROWSER_NewDriverURL]
-	#forceref $sBrowserVersion, $bFlag64
```

Comment:
Sorry, I simple don't understand why this is removed?

Reply:
Because it is used
https://github.com/mlipok/au3WebDriver/blob/WinHttp.WinHttpRequest.5.1/wd_helper.au3#L3607
https://github.com/Danp2/au3WebDriver/blob/b22f64fdba4dc0d74d93bb28a31e87f2f08bf755/wd_helper.au3#L3570


thus there is no need to use `#forceref to prevent warnings where parameters are mandatory (such as in Windows message handlers) but are not used inside the function`

Reply:
Ah, understood. Thanks @mlipok .